documentation/AuthoringSolutionsTutorial.md

```diff
@@ -0,0 +1,342 @@
+                        }
+                    },
+                    "cloud4allVoiceProfile-GlobalContext\\.Speed": {
+                        "transform": {
+                            "type": "fluid.transforms.linearScale",
+                            "valuePath": "http://registry\\.gpii\\.net/common/speechRate",
+                            "factor": 0.125,
+                            "offset": -12.125
+                        }
+                    }
+                }
+            }
+        ],
+
+```
+
+The first item is the type, for us it's  `gpii.settingsHandlers.INISettingsHandler`, because JAWS stores it's settings in an INI file. There are other settings handlers for XML Files, using the Windows Registry, and other places across platforms.
+
+TODO insert link to desc of other settings handler types
+
+Next up is an `options` block of JSON. The keys that are available in here will differ depending upon which settings handler is being used. In the case of the `INISettingsHandler` we have the option `filename` that is the path to the INI file containing the settings.
+
+The filename here has some interesting characteristics.
+
+```json
+"filename": "${{environment}.APPDATA}\\Freedom Scientific\\JAWS\\15.0\\Settings\\VoiceProfiles.ini"
+```
+
+First off, because this is in a JSON string we do have to remember about standard escape sequences, so in order to write a single backslash, you need to put 2 in a row.
+
+Also, take a look at `${{environment}.APPDATA}`.  Inside our document processing, we support certain special variables that can be used in strings. In this case, the `environment` variable allows us to access any variable in the standard windows command line environment.  On windows `APPDATA` is a standard variable that points to a directory on the machine that contains data for various applications.  
+
+TODO Provide full path example of what the path above resolves to when I get back to a windows machine.
+
+TODO Point to further/future documentation that says what string variables are available in which fields.
+
+#### Capabilities
+
+The next two parts of the settings handler require a bit more overview and explanation. These are the `capabilities` and `capabilitiesTransformations` sections.
+
+The `capabilities` section, true to it's name, lists the capabilities of your solution. But what does that mean practically? Typically the items here fall in to 2 categories. First, you'll usually specify a capabilitiy that is really just the name, or id, of your solution. So if a user always uses JAWS on their other machines, and wants to use JAWS specifically rather than another screenreader, they can specify that. But the capabilities can also list more generic terms that are often found in the common terms list. ( You can see an example of most of our common terms in `universal/testData/ontologies/ISO24751-flat.json` ) For example, if your application was an onscreen keyboard, you could list `control.onscreenKeyboard`.
```

Comment:
Should mention that "capabilities" and "capabilitiesTransformations", in terms of their matchmaking implications, are somewhat on an even footing - it's most likely that a matchmaker that we actually deploy, once it is actually working, will account for the goodness of match of a solution in terms of how many of the user's preferences it can satisfy. And one way of accounting for how to satisfy a user's preference is to have a transform encoding the mapping from it to the solution's settings. Now in the case there is no direct preference of this kind, we would like to still be able to account for the fact that a solution might meet a need of the user somehow. This is what we do with the "capabilities" entries.